pkgs/by-name/co/codesnap/package.nix

```
@@ -0,0 +1,33 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "codesnap";
+  version = "0.8.2";
+
+  src = fetchFromGitHub {
+    owner = "mistricky";
+    repo = "CodeSnap";
+    rev = "refs/tags/v${version}";
```

```suggestion
    tag = "v${version}";
```
Recently introduced and better.

---

Thanks!